kubectl-pg/cmd/util.go

```diff
@@ -0,0 +1,31 @@
+package cmd
+
+import (
+	"flag"
+	"k8s.io/client-go/tools/clientcmd"
+	"os/exec"
+	"path/filepath"
+	"k8s.io/client-go/util/homedir"
+	restclient "k8s.io/client-go/rest"
+)
+
+func getConfig()(*restclient.Config){
+	var kubeconfig *string
+	if home := homedir.HomeDir(); home != "" {
+		kubeconfig = flag.String("kubeconfig", filepath.Join(home, ".kube", "config"), "(optional) absolute path to the kubeconfig file")
+	} else {
+		kubeconfig = flag.String("kubeconfig", "", "absolute path to the kubeconfig file")
+	}
+	flag.Parse()
+
+	config, err := clientcmd.BuildConfigFromFlags("", *kubeconfig)
+	if err != nil {
+		panic(err)
+	}
+	return config
+}
+
+func getCurrentNamespace() (string){
+	currentNamespace, _ := exec.Command("kubectl", "config", "view", "--minify", "--output", "jsonpath={..namespace}").CombinedOutput()
```

Comment:
If `default` namespace is used, this command won't return anything which leads to panic:
```panic: an empty namespace may not be set when a resource name is provided```

In this case `currentNamespace` should be set to `default`.

Reply:
```\zalando\postgres-operator\kubectl-pg> kubectl config view --minify --output jsonpath={..namespace}```
```default```

Irrespective of the current namespace it returns us the namespace including default namespace which is working in my case. As I mentioned above. 
Let me know if my understanding is wrong or if am missing anything. Thanks! :)

Reply:
when I start minikube the current context does not contain a namespace. So it's possible that it can be empty.

Reply:
Okay, by default I will set it to ```default```